Vagrantfile

```diff
@@ -20,14 +21,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  config.ssh.insert_key = false # for vagrant 1.8.5 see https://github.com/mitchellh/vagrant/issues/7610#issuecomment-234609660

```

Comment:
@smartbit i faced this issue yesterday & i fix it by applying a patch  from this link https://github.com/mitchellh/vagrant/issues/7610